src/routers/resource_router.py

```diff
@@ -246,6 +255,7 @@ def get_resources(
         resource_filters: ResourceFilters,
         user: KeycloakUser | None = None,
         platform: str | None = None,
+        get_image: bool = False,
```

Comment:
```suggestion
        get_image: bool = False,
```
Similarly, we can move this down to the `Organisation` router. As is, now every endpoint, including e.g., datasets, will have this `get_image` parameter even though images aren't supported for those types. That would be confusing for the user.